regression/cbmc-cover/branch-loop1/test.desc

```diff
@@ -1,6 +1,6 @@
 CORE
 main.c
---xml-ui --cover branch
+--xml-ui --cover branch --trace
```

Comment:
Seems like this is broken somehow?

Reply:
Yes, it was broken as the code generating XML output is used for both trace and test-suite output. I have therefore changed the patch to introduce a new command-line option "show-test-suite."